jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
Why jest? Should prefer vitest. It's faster and less quirky, supports ts and esm out of the box